tests/plugins/formatting.test.ts

```diff
@@ -561,4 +561,12 @@ describe("Autoresize", () => {
     model.dispatch("AUTORESIZE_COLUMNS", { sheetId, cols: [0] });
     expect(model.getters.getColSize(sheetId, 0)).toBe(initialSize);
   });
+
+  test("Test to verify row height calculation for cells with formula syntax", async () => {
+    const initialSize = model.getters.getRowSize(sheetId, 0);
+    setCellContent(model, "A2", "10");
+    setCellContent(model, "A1", "=A2\n\n+5\n\n-3\n\n-9");
```

Comment:
you don't any reference to show this behavior. Let's keep it simple and not add the reference
```suggestion
    setCellContent(model, "A1", "=5\n\n-3\n\n-9");
```